### ISSUE TITLE
Redux devtools annotations not being generated

### ISSUE THREAD

**Issue by bhackett1024** [replayio/gecko-dev#843]
The latest gecko-dev release isn't generating redux devtools annotations.  I tried looking into this, but after building the browser locally I wasn't able to get it to generate annotations either.  I was loading the library view at https://app.replay.io, which I understand should be able to attach to the redux devtools backend and generate these annotations.

I applied this [patch](https://gist.github.com/bhackett1024/dce6801eaa54962ac660b25ec3213312) and tried again.  Looking at the terminal output while recording this showed that we were initializing the devtools (`REDUX_DEVTOOLS_INITIALIZE`) and installing the devtools extension on the window (`INSTALL_REDUX_DEVTOOLS_EXTENSION`) but none of the properties on the extension were being called (`EXTENSION_*`) and no messages were being sent which would create the annotations (`REDUX_DEVTOOLS_SEND_MESSAGE`).  I don't know enough about the interface between redux and the devtools backend to investigate further.

**Comment by gideonred** [comment]
cc @markerikson

**Comment by markerikson** [comment]
Mmm... I don't think this was a valid test, actually.

We _do_ create the initial Redux store in our `<App>` component, which should indeed be rendered on the library page.

However, I have the store configured to only connect to the Redux DevTools in dev builds, so you wouldn't see it in the prod app.

I've been using two CodeSandboxes as my sample pages:

- Redux: https://un8my.csb.app/
- Jotai: https://5z1mjb.csb.app/

**Comment by bhackett1024** [comment]
OK, thanks.  Testing on https://un8my.csb.app/, I now see redux devtools annotations being generated in both my local build *and* the latest released build.

```
bhackett@Brians-MacBook-Pro gecko-dev % RECORD_REPLAY_VERBOSE=1 RECORD_REPLAY_DUMP_ANNOTATIONS=1 /Applications/Replay.app/Contents/MacOS/replay https://un8my.csb.app
...
[10150/15941867834970882012 1651872542.734] Attach BuildId macOS-gecko-20220504-4a46d8d2767e-177ace34c9dc Version linker-macOS-8783-177ace34c9dc
...
[10150/1 5.161] RecordReplayOnAnnotation redux-devtools-bridge {"type":"INIT_INSTANCE","source":"@devtools-page","instanceId":1}
...
[10150/1 5.292] RecordReplayOnAnnotation redux-devtools-bridge {"type":"STATE","payload":{"monitorState":{},"nextActionId":1,"stagedActionIds":[0],"skippedActionIds":[],"currentStateIndex":0,"isLocked":false,"isPaused":false},"source":"@devtools-page","instanceId":1,"libConfig":{"name":"React Redux App","serialize":false,"type":"redux"},"actionsById":"{\"0\":{\"type\":\"PERFORM_ACTION\",\"action\":{\"type\":\"@@INIT\"},\"timestamp\":1651872547895}}","computedStates":"[{\"state\":{\"counter\":{\"value\":0}}}]","committedState":false}
```

**Comment by markerikson** [comment]
Well, _that_ seems like a good sign...

Let me just try pulling down the latest prod build and make a new recording and see what happens?

**Comment by bhackett1024** [comment]
Sure, note though that I didn't try to use the redux devtools, but if you are able to log the annotation output above to a file then you'll see directly what annotations are being added to the new recording.  In the past, on windows I've had to pipe the browser output to a file first though in order to see it.

**Comment by markerikson** [comment]
Update:

**IT LOOKS LIKE THIS ALL WORKS!**

We have confirmation of it working on Jason's Mac, and on my Windows machine _with a freshly downloaded browser_.  Where I was particularly seeing nothing happening was when I did the recordings with an in-place upgrade of an existing Replay browser folder on Windows.  (Yes, I did actually stop and start that browser. Multiple times.)  

So, not sure what's up with the in-place upgraded build, but it seems like this really is all working as intended now!
